common/config/jaeger.go

```diff
@@ -26,32 +30,76 @@ func (j *Jaeger) DestType() common.DestinationType {
 }
 
 func (j *Jaeger) ModifyConfig(dest ExporterConfigurer, currentConfig *Config) error {
+	config := dest.GetConfig()
+	uniqueUri := "jaeger-" + dest.GetID()
 
-	if !isTracingEnabled(dest) {
-		return ErrorJaegerTracingDisabled
+	url, urlExists := config[JaegerUrlKey]
+	if !urlExists {
+		return ErrorJaegerMissingURL
 	}
 
-	url, urlExist := dest.GetConfig()[JaegerUrlKey]
-	if !urlExist {
-		return ErrorJaegerMissingURL
+	var exporterName string
+	var exporterConfig GenericMap
+
+	certPem, certExists := dest.GetConfig()[JaegerCertPemKey]
+	keyPem, keyExists := dest.GetConfig()[JaegerKeyPemKey]
+	if certExists && keyExists {
```

Comment:
these parameters are not relevant for exporters. they should be removed and the tls should b e set based on the url scheme or a dedicated parameter